modfinder/README.md

```diff
@@ -0,0 +1,55 @@
+The ModFinder Module of BioJava
+=====================================================
+
+A tutorial for the modfinder module of [BioJava](http://www.biojava.org)
+
+## About
+<table>
+    <tr>
+        <td>
+            <img src='https://cloud.githubusercontent.com/assets/840895/22190971/fe5cd304-e0f4-11e6-9eb5-c1b071312081.png'>
+        </td>
+        <td>
+            The <i>modfinder</i> module of BioJava provides an API The <i>alignment</i> module of BioJava provides an API for identification of protein pre-, co-, and post-translational modifications from structures.
```

Comment:
The phrase _The alignment module of BioJava provides an API_ should be deleted here.